### ISSUE TITLE
Image expand not working correctly.

### ISSUE THREAD

**Issue by lemmygod** [howdy-tsc/LemmyTools#14]
Video recording:

https://github.com/howdy-tsc/LemmyTools/assets/137420192/aa061e3f-264f-4629-80ee-0c08730aec8e

**Comment by howdy-tsc** [comment]
Thank you so much for the video. That is very helpful. For an amateur programmer creating this functionality from scratch was a challenge so this doesn't surprise me. I saw the same (click and drag) issue where it would immediately expand to as large as possible with barely any movement while developing and thought I had worked it out. Few questions: what was the auto expand size in the settings set to? What resolution of screen? Also, What happens if you enable the "hide Lemmy sidebars" and then try. Also does it seem to work better with the old.reddit restyle on? The issue I was having is with the default Lemmy styling there is barely any room to expand to and the click and drag is getting expanded immediately. I have a speed multiplier in the code that could be changed as well. Perhaps you have mouse sensitivity really high? 

Thanks again.
